test/functional/grpcServer/handlers/core/getTransactionHandlerFactory.js

```diff
@@ -0,0 +1,74 @@
+const startDapi = require('@dashevo/dp-services-ctl/lib/services/startDapi');
+
+const {
+  Transaction,
+  PrivateKey,
+} = require('@dashevo/dashcore-lib');
+
+describe('getTransactionHandlerFactor', function main() {
+  this.timeout(260000);
+
+  let removeDapi;
+  let dapiClient;
+  let transaction;
+  let transactionId;
+
+  beforeEach(async () => {
+    const {
+      dapiCore,
+      dashCore,
+      remove,
+    } = await startDapi();
+
+    removeDapi = remove;
+
+    dapiClient = dapiCore.getApi();
+
+    const coreAPI = dashCore.getApi();
+
+    await coreAPI.generate(1000);
+
+    const { result: fromAddress } = await coreAPI.getNewAddress();
+    const { result: privateKeyString } = await coreAPI.dumpPrivKey(fromAddress);
+    const { result: toAddress } = await coreAPI.getNewAddress();
+
+    const privateKey = new PrivateKey(privateKeyString);
+    await coreAPI.generate(500);
+    await coreAPI.sendToAddress(fromAddress, 10);
+    await coreAPI.generate(10);
+
+    const { items: unspent } = await dapiClient.getUTXO(fromAddress);
+
+    const amount = 10000;
+
+    transaction = new Transaction();
+
+    transaction.from(unspent)
+      .to(toAddress, amount)
+      .change(fromAddress)
+      .fee(668)
+      .sign(privateKey);
+
+    ({ result: transactionId } = await coreAPI.sendRawTransaction(transaction.serialize()));
+  });
+
+  afterEach(async () => {
+    await removeDapi();
+  });
+
+  it('should respond with a transaction by it\'s ID', async () => {
+    const result = await dapiClient.getTransaction(transactionId);
+    const receivedTx = new Transaction(Buffer.from(result));
+    expect(receivedTx.toJSON()).to.deep.equal(transaction.toJSON());
```

Comment:
expect(Buffer.from(result).toString('hex')).to.deep.equal(transaction.serialize());